packages/cashier/src/pages/account-transfer/account-transfer-no-account/account-transfer-no-account.tsx

```diff
@@ -47,15 +39,11 @@ const AccountTransferNoAccount = observer(({ openAccountSwitcherModal }: TAccoun
                 className='account-transfer-no-account__button'
                 primary
                 large
-                onClick={() =>
-                    openAccountSwitcherModal
-                        ? openAccountSwitcherModal()
-                        : is_pre_appstore
-                        ? openModal('currency_selection')
-                        : toggleAccountsDialog()
-                }
+                onClick={() => {
+                    history.push(routes.traders_hub);
+                }}
             >
-                <Localize i18n_default_text='Create account' />
+                <Localize i18n_default_text={`Back to trader's hub`} />
```

Comment:
@rostislav-deriv we cannot use backticks for now in deriv-app, pls check this [link](https://deriv-group.slack.com/archives/C3AHMSM3P/p1677553396124379)
```suggestion
                <Localize i18n_default_text="Back to trader's hub" />
```